### PR TITLE
fixed an error in method collect_unknown_message

### PR DESCRIPTION
**Pull request by voith**
please have a look at diff https://github.com/scrapinghub/frontera/commit/d2808841c8bd93e495bf8ba39622afea1369e639#diff-4ccfaf7c9b69baf8969a4daab4c21f22L148 to understand the change that I've made

### CODE REVIEW

**Comment by sibiryakov** [comment]
@voith this is absolutely wrong. Only logging line needs to be fixed here, the msg have to be propagated, not it's type. The purpose of this method is to customize it in subclasses, respectively handling custom messages.

**Comment by voith** [comment]
@sibiryakov will fix it

**Comment by voith** [comment]
fixed!

File: frontera/worker/strategy.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __init__(self, settings, strategy_class):
         logger.info("Strategy worker is initialized and consuming partition %d", partition_id)
 
     def collect_unknown_message(self, msg):
-        logger.debug('Unknown message type %s', type)
+        logger.debug('Unknown message type %s', msg)
```
Comment (sibiryakov) [inline frontera/worker/strategy.py]: let's remove word "type" from the logging message.

Comment (voith) [inline frontera/worker/strategy.py]: done

**Comment by sibiryakov** [comment]
thanks!

**Pull request merged**